environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
   - python=3.8 # nail the python version, so conda does not try upgrading / downgrading
   - conda-forge::ctapipe=0.12
+  - numpy~=1.22.4 # higher versions >=1.23 don't work due to astropy4 dependency in ctapipe0.12
```

Comment:
Hi @mdpunch ,

As noted already by @maxnoe , the syntax for the environment specifications for `conda` does not follow the same as `setup.py`.
Please change:
```  - numpy~=1.22.4```
with:
```  - numpy=1.22```

Reply:
Done (I think)